lib/workers/global/initialize.ts

```diff
@@ -29,6 +30,7 @@ async function setDirectories(input: AllConfig): Promise<AllConfig> {
     logger.debug('Using cacheDir: ' + config.cacheDir);
   }
   await fs.ensureDir(config.cacheDir);
+  GlobalConfig.set({ ...GlobalConfig.get(), cacheDir: config.cacheDir });
```

Comment:
why do we need this? 🤔 why we don't need it for file cache?

Reply:
Would you ask me to use `util/fs` functions for file existence check, if I used `fs.stat()` directly? 🙂

But the problem with `util/fs` functions is that `cacheDir` should be initialized in `GlobalConfig`, which isn't the case at the moment of cache init.

Reply:
yes, let's use fs-extra directly in cache init to simplify things.

we can refactor global init later to be usable inside cache.